tests/test_dbt.py

```diff
@@ -80,20 +81,21 @@ def test_dbt_glaredb_external_postgres(
             """
         )
 
-    with tests.tools.env("DBT_USER", glaredb_connection.info.user):
-        res: dbtRunnerResult = dbtRunner().invoke(
-            [
-                "run",
-                "--project-dir",
-                dbt_project_path,
-                "--profiles-dir",
-                dbt_project_path,
-                "-m",
-                model_name,
-            ]
-        )
-
-        assert res.success is True
+    with tests.tools.env("GLAREDB_PORT", str(glaredb_connection.info.port)):
+        with tests.tools.env("DBT_USER", glaredb_connection.info.user):
+            res: dbtRunnerResult = dbtRunner().invoke(
+                [
+                    "run",
+                    "--project-dir",
+                    dbt_project_path,
+                    "--profiles-dir",
+                    dbt_project_path,
+                    "-m",
+                    model_name,
+                ]
+            )
+
+            assert res.success is True
```

Comment:
The nested `with` block looks kind of gnarly. [Maybe something like this instead?](https://stackoverflow.com/a/46481946)

Reply:
huh! amazing!